src/BenchmarksApps/TechEmpower/PlatformBenchmarks/Data/RawDb.cs

```diff
@@ -203,7 +210,33 @@ public async Task<World[]> LoadMultipleUpdatesRows(int count)
                     await reader.NextResultAsync();
                 }
             }
+
+            using (var connection = await _updateDataSource.OpenConnectionAsync())
+            using (var updateCmd = new NpgsqlCommand(BatchUpdateString.Query(count), connection))
```

Comment:
Might as well just call `_updateDataSource.CreateCommand` (and the same thing above for batch).

Reply:
You're right. I prefer to do it separately/later though, to cleanly isolate the changes in this PR from anything else (so we hopefully get a clean perf change reading).

I've made a note and will do this later.